### PR TITLE
Added info for compiling on RPM based distros

### PR DESCRIPTION
**Pull request by ssbarnea**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -66,6 +66,14 @@ you can install the =vobsub2srt= package:
 You can build a *.deb package (Debian/Ubuntu) with =make package=.  The package
 is created in the =build= directory.
 
+*** .rpm (Centos/RHEL/Fedora)
+#+BEGIN_EXAMPLE
+  sudo yum install -y cmake tesseract-devel dpkg libtiff-devel libwebp-devel libpng-devel giflib-devel
+  ./configure -DBUILD_STATIC=ON
```
Comment (ruediger) [inline README.org]: BUILD_STATIS is a hack and shouldn't be encouraged.

**Comment by ruediger** [comment]
In #58 you say that it's not working?

**Comment by glensc** [comment]
why `dpkg` as dependency? perhaps there's other problem that needs to be fixed outside this PR?

**Comment by ssbarnea** [comment]
Not used or needing the tool anymore.

**Pull request closed without merging**